aten/src/ATen/WrapDimUtils.h

```diff
@@ -30,14 +30,15 @@ static inline int64_t maybe_wrap_dim(int64_t dim, const std::vector<std::vector<
   return maybe_wrap_dim(dim, tensor_sizes[0].size());
 }
 
-// wrap each of dims basing on dim_post_expr
-static inline void maybe_wrap_dims(std::vector<int64_t>& dims, int64_t dim_post_expr) {
+static inline void maybe_wrap_dims_n(int64_t * dims, int64_t ndims, int64_t dim_post_expr) {
+  // wrap each of dims basing on dim_post_expr
```

Comment:
Can we improve this comment? I'm not sure what "basing on dim_post_expr" means.

Reply:
Reworded to "taking dim_post_expr as the true number of dimensions".